Client/mods/deathmatch/logic/CClientTeam.h

```diff
@@ -44,6 +44,7 @@ class CClientTeam final : public CClientEntity
     void AddPlayer(CClientPlayer* pPlayer, bool bChangePlayer = false);
     void RemovePlayer(CClientPlayer* pPlayer, bool bChangePlayer = false);
     void RemoveAll();
+    std::vector<CClientPlayer*> GetPlayers() noexcept;
```

Comment:
Actually, this method can throw an exception(`std::bad_alloc`). So please, remove `noexcept` and add `const`.